yb-voyager/cmd/importData.go

```diff
@@ -122,42 +122,31 @@ func startFallforwardSynchronizeIfRequired(tableList []string) {
 		"--source-db-host", tconf.Host,
 		"--source-db-port", fmt.Sprintf("%d", tconf.Port),
 		"--source-db-user", tconf.User,
-		"--source-db-password", fmt.Sprintf("'%s'", tconf.Password),
 		"--source-db-name", tconf.DBName,
 		"--source-db-schema", tconf.Schema,
 		"--table-list", strings.Join(tableList, ","),
 		fmt.Sprintf("--send-diagnostics=%t", callhome.SendDiagnostics),
 	}
-	fallForwardSynchronizeCmdStr := strings.Join(fallForwardSynchronizeCmd, " ")
+
 	if utils.DoNotPrompt {
 		fallForwardSynchronizeCmd = append(fallForwardSynchronizeCmd, "--yes")
 	}
 	if disablePb {
 		fallForwardSynchronizeCmd = append(fallForwardSynchronizeCmd, "--disable-pb")
 	}
+	fallForwardSynchronizeCmdStr := "SOURCE_DB_PASSWORD=*** " + strings.Join(fallForwardSynchronizeCmd, " ")
 
-	utils.PrintAndLog("Starting ff synchronize with command %s", fallForwardSynchronizeCmdStr)
-	binary, lookErr := exec.LookPath("yb-voyager")
+	utils.PrintAndLog("Starting fall-forward synchronize with command:\n %s", color.GreenString(fallForwardSynchronizeCmdStr))
+	binary, lookErr := exec.LookPath(os.Args[0])
```

Comment:
Should we be using `AbsPath()`?

Reply:
LookPath returns the abs path. 
check https://github.com/yugabyte/yb-voyager/pull/1036#discussion_r1314695347